client/components/Coverages/CoverageIcons.tsx

```diff
@@ -0,0 +1,260 @@
+import * as React from 'react';
+import moment from 'moment-timezone';
+import {getUserInitials} from './../../components/UserAvatar';
+import * as config from 'appConfig';
+import {IPlanningCoverageItem, IG2ContentType, IContactItem, IPlanningConfig} from '../../interfaces';
+import {IUser, IDesk} from 'superdesk-api';
+import {gettext} from 'superdesk-core/scripts/core/utils';
```

Comment:
`gettext` should be used from the `superdeskApi.localization` interface

Reply:
Good catch. I'll update it. I didn't import this myself - simply kept it from the previous component which I have refactored :upside_down_face: